articles/cpptod.dd

```diff
@@ -560,7 +560,7 @@ $(H4 The D Way)
 
         The D version is analogous, though a little simpler, taking
         advantage of
-        $(HTTPS dlang.org/spec/template.html#implicit_template_properties,
+        $(DDSUBLINK spec/template, implicit_template_properties,
```

Comment:
Any specific reason why you sometimes use LINK2 and sometimes DDSUBLINK?

Reply:
When it referes to a sublink in a page, I use `DDSUBLINK`. When it refers to the whole page, I use `LINK2`. I saw this commit, [use DDSUBLINK instead of LINK2 ](https://github.com/dlang/druntime/commit/ea4785d189ca7303e034e6c392f9b8fde31faef7) in druntime repo in the past, I thought it would be prefered.